### PR TITLE
updated  setup links in all READMEs

### LINKED ISSUE #370: Change installation link in exercise READMEs
Each [exercise](https://github.com/exercism/ecmascript/tree/master/exercises) has a [README file with a setup link in it](https://github.com/exercism/ecmascript/blob/master/exercises/accumulate/README.md#setup).

The link points to http://exercism.io/languages/ecmascript/ but the setup instructions are in http://exercism.io/languages/ecmascript/installation. 

**Task:** Change the link to the correct installation link in README files of `all` exercises.

### PR DESCRIPTION
**Pull request by lantran**
Updated all setup links from `http://exercism.io/languages/ecmascript` to `http://exercism.io/languages/ecmascript/installation`

Closes #370

### CODE REVIEW

File: exercises/collatz-conjecture/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Resulting in 9 steps. So for input n = 12, the return value would be 9.
 Go through the setup instructions for ECMAScript to
 install the necessary dependencies:
 
-[http://exercism.io/languages/ecmascript](http://exercism.io/languages/ecmascript)
+[http://exercism.io/languages/ecmascript/installation](http://exercism.io/languages/ecmascript)
```
Comment (tejasbubane) [inline exercises/collatz-conjecture/README.md]: Can you make this consistent with all other files?

**Comment by tejasbubane** [comment]
Thanks @lantran Thanks was fast 🚀 

Just one small change and then we should be good to go!

**Comment by lantran** [comment]
@tejasbubane : updated

**Comment by tejasbubane** [comment]
Thanks @lantran and congrats on your first contribution to exercism! 🎉

**Pull request merged**